Timelines/Timeline_onScroll_Animation_1/app.js

```diff
@@ -0,0 +1,19 @@
+window.addEventListener('scroll', () => {
```

Comment:
Try adding comments wherever possible so that it makes your code flow clear to the reader.

Reply:
Ok I will try to add comments